### DISCUSSION TITLE
nao  entendi esse problema

### DISCUSSION THREAD

**Discussion by lucccansousa** [balta-io/forum#331] [Tira Dúvidas]
![image](https://github.com/balta-io/forum/assets/131711446/6a93cf59-e3e2-4cba-b139-bfc07bf28fb5)
nao eentendi oq eu fiz de errado

**Comment by lucascodecraft** [comment]
> ![image](https://user-images.githubusercontent.com/131711446/238702556-6a93cf59-e3e2-4cba-b139-bfc07bf28fb5.png) nao eentendi oq eu fiz de errado
Esse error está acontecendo porque o localStorage.getItem('todos') é null e você esta tentando passar esse valor null para um metodo que espera uma string.

Como sugestão tente isso:
load() {
const data = localStorage.getItem('todos');
if (data !== null) {
this.todos = JSON.parse(data);
}
}

ou

load() {
const data = localStorage.getItem('todos');
this.todos = data !== null ? JSON.parse(data) : [];
}
